doc/melange.rst

```diff
@@ -57,8 +57,10 @@ Finally, add a source file to build:
 
   echo 'Js.log "hello from melange"' > hello.ml
 
-After running ``dune build output/hello.js``, Dune produces the following
-file structure:
+After running ``dune build output/hello.js`` or ``dune build @melange``
+(by default, all ``melange.emit`` stanzas will attach their targets to
+the  ``melange`` alias),
```

Comment:
This behavior is explained in the `alias` field of the `melange.emit` stanza docs, a few paragraphs below. I am not sure it's worth adding a reference?

Reply:
When I was following the introduction section and got `hello.js` built, I started to hacking the minimal example around and was met with surprises when `dune build` (without specifying the `melange` alias) wasn't generating the javascript files.

I personally wasted some decent time trying to debug it was the case and only realized such behavior after very carefully reading the document again.

Yes the behavior is already documented, but I think it's probably not be only me getting the impression that the targets added by `melange.emit` stanza will be attached to the `default` alias (as for most other dune stanzas) after just following the current introduction.

Reply:
I was not aware that all other targets (including jsoo ones, it seems) are added to the default alias. I am onboard with adding `melange.emit` targets to the default alias, but I'm not sure if this addition was avoided for some reason that I'm not aware of. @rgrinberg @anmonteiro do you know?

(Probably something that should be discussed in a separate issue)

Reply:
I think adding it to the default alias (`@all` in this case) is the correct thing to do.